docs/guest.md

```diff
@@ -0,0 +1,40 @@
+# Tanzu Kubernetes Grid Service
+This document discusses the velero vSphere plugin installation process in a Tanzu Kubernetes Grid Service environment
+aka as Guest Cluster. For more information on Tanzu Kubernetes Grid Service refer to their [documentation](https://docs.vmware.com/en/VMware-Tanzu-Kubernetes-Grid/index.html).
+
+## Compatibility
+| vSphere/ESXi Version       | vSphere CSI Version                                                | Kubernetes Version | Velero Version    | Velero Plugin for vSphere Version |
+|----------------------------|--------------------------------------------------------------------|--------------------|-------------------|-----------------------------------|
+| vSphere 7.0P02/ESXi 7.0P02 | Driver version is not v2.1.0, but feature is equivalent to v2.1.0. | v1.16-v1.19        | v1.5.1 and higher | v1.1.0 and higher                 |
+
+## Prerequisites
+* Prior to the installation of Velero vSphere plugin in Guest cluster it is necessary to have Velero and Velero 
+vSphere plugin installed in the Supervisor Cluster, please refer to [Supervisor Cluster Velero vSphere Plugin](docs/supervisor.md)
+installation guide.
+* DataManager needs to be installed, please refer [DataManager Documentation](docs/supervisor-datamgr.md)
+
+## Install
+Please refer [Vanilla Installation Guide](docs/vanilla.md)
+
+* The installation of Velero vSphere Plugin in guest cluster may take over ten minutes to complete, this is primarily
+because the plugin waits for the velero app operator to write the Secret into a predefined namespace.
+* The installation may hang forever if Velero is not installed in the supervisor cluster.
+
+## Uninstall
+Please refer [Vanilla Uninstall Guide](docs/vanilla.md)
+
+## Backup
+Please refer [Vanilla Backup Guide](docs/vanilla.md)
+
+On Backup, a Snapshot CRD is created to track the vSphere volume snapshot operation and data upload.
```

Comment:
Add an example snapshot CRD.  Can be added in a followup PR later.

Reply:
Done, resolved other comments as well.